demo/index.ts

```diff
@@ -13,7 +13,26 @@ import {
 
 import { Godrays, defaultConfig } from "../dist/index";
 
-console.log(Godrays);
+var raf = window.requestAnimationFrame;
+var skipProcent = 0.9;
+var rafs = 0;
+
+window.requestAnimationFrame = (callback: FrameRequestCallback): number => {
```

Comment:
I know it's only for the demo, but it's still better not to change the method on `window`. Why didn't you create a function that calls `window.requestAnimationFrame`? If someone wants to play with the code in your demo, they may try to use `window.requestAnimationFrame` with unexpected results.